redundancy/common/redundancyHandler.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+import (
+	"errors"
+	"fmt"
+)
+
+const minRoundsOfInactivity = 2 // the system does not work as expected with the value of 1
```

Comment:
Technically, it can work with the value of 2. Maybe someone wants to set it to 2 as the backups will kick in more faster. I would keep it set to 2 but the config value from the .toml file set to 3.

Reply:
ok